Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml

```diff
@@ -508,196 +549,113 @@ tasks:
     message:
       to:
       subject:
-        simple: Select Remediation Action
       body:
-        simple: Cortex Xpanse has the ability to fully remediate or augment your remediation process via integrations in other security and IT tools and control surfaces. The options presented in this prompt may vary depending on the integrations that your organization has configured and the type of alert. You can learn more about why you're seeing these options and what they mean at https://pan.dev.
       methods: []
-      format: html
+      format: ""
```

Comment:
HTML format is not needed to be specified anymore?

Reply:
This GIT diff is really confusing and not sure why it is comparing these two tasks that are totally different.  All of the "Select Remediation Action" data collection tasks have been moved to "Cortex ASM - Remediation Path Rules" as a single task and it is using `format: html` 
https://github.com/johnnywilkes/content-jw/blob/14a3c5b116527d9cab0277e50686ab9c06c0a26f/Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Remediation_Path_Rules.yml#L1417